python/src/trezorlib/messages.py

```
@@ -10967,6 +10997,44 @@ def __init__(
 
 
+class TronCancelAllUnfreezeV2Contract(protobuf.MessageType):
+    MESSAGE_WIRE_TYPE = None
+
+
+class TronVoteWitnessContract(protobuf.MessageType):
+    MESSAGE_WIRE_TYPE = None
+    FIELDS = {
+        2: protobuf.Field("votes", "Vote", repeated=True, required=False),
+        3: protobuf.Field("support", "bool", repeated=False, required=False),
+    }
```

Ruff (0.14.5)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)